lib/Rex/Group/Lookup/XML.pm

```diff
@@ -33,7 +33,7 @@ use Rex -base;
 require Exporter;
 use base qw(Exporter);
 use vars qw(@EXPORT);
-XML::LibXML->require;
+use XML::LibXML;
```

Comment:
This is right, but we can't take care for these old distributions. CentOS 4 support ended 2012 (https://de.wikipedia.org/wiki/CentOS#Versionen) and RHEL 4 support ended 2015 (https://de.wikipedia.org/wiki/Red_Hat_Enterprise_Linux#Versionen)